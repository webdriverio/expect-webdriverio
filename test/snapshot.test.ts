import { test, expect } from 'vitest'
import { expect as expectExport } from '../src/index.js'

test('supports snapshot testing', () => {
    const exp = expectExport;
    expect(exp).toBeDefined()
    expect(exp({}).toMatchSnapshot).toBeDefined()
    exp({ a: 'a' }).toMatchSnapshot()
})
