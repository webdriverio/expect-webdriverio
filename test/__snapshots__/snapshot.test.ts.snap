// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`test/snapshot.test.ts > supports snapshot testing 1`] = `
{
  "a": "a",
}
`;
